EmbeddedPkg/EmbeddedPkg.dsc

```diff
@@ -227,17 +229,23 @@
       TimerLib|MdePkg/Library/BaseTimerLibNullTemplate/BaseTimerLibNullTemplate.inf
   }
 
-  EmbeddedPkg/Universal/MmcDxe/MmcDxe.inf
+  # EmbeddedPkg/Universal/MmcDxe/MmcDxe.inf     # MU_CHANGE - Too many conversion errors to trust.
 
   EmbeddedPkg/Library/AcpiLib/AcpiLib.inf
   EmbeddedPkg/Library/DebugAgentTimerLibNull/DebugAgentTimerLibNull.inf
+!if $(TOOL_CHAIN_TAG) == GCC5     # MU_CHANGE - Too many external errors to deal with.
   EmbeddedPkg/Library/FdtLib/FdtLib.inf
+!endif                            # MU_CHANGE
   EmbeddedPkg/Library/PrePiHobLib/PrePiHobLib.inf
   EmbeddedPkg/Library/PrePiMemoryAllocationLib/PrePiMemoryAllocationLib.inf
 
+!if $(TOOL_CHAIN_TAG) == GCC5     # MU_CHANGE - Requires FdtLib, which is also broken.
   EmbeddedPkg/Drivers/ConsolePrefDxe/ConsolePrefDxe.inf
+!endif                            # MU_CHANGE
   EmbeddedPkg/Drivers/DtPlatformDxe/DtPlatformDxe.inf
+!if $(TOOL_CHAIN_TAG) == GCC5     # MU_CHANGE - Requires FdtLib, which is also broken.
   EmbeddedPkg/Drivers/FdtClientDxe/FdtClientDxe.inf
+!endif                            # MU_CHANGE

```

Comment:
Out of curiosity, are these still broken on VS2022? It is somewhat concerning that we just don't build them but our consumers could still try to pull them and use them (theoretically it works for GCC).

Reply:
Here are a small number of examples from FdtLib:

fdt_strtoul.c(36): 'return': conversion from 'UINTN' to 'unsigned long'
return': conversion from 'INTN' to 'int
fdt_overlay.c(613): '=': conversion from '__int64' to 'uint32_t
fdt.c(93): '<': signed/unsigned mismatch
libfdt_env.h(83): 'function': conversion from 'int' to 'UINT8'


It looks like it is all fixable, but would take time to validate.

Reply:
I'm good to file an issue to come back to this and check it in as is.

Reply:
Issue is logged here:

https://github.com/microsoft/mu_tiano_plus/issues/309